onnxscript/tests/function_libs/torch_lib/ops_test.py

```diff
@@ -90,9 +90,10 @@ def _split_function_and_wrangler(
 # according to https://pytorch.org/docs/stable/testing.html
 OPINFO_PRECISION_TABLE: dict[torch.dtype, tuple[float, float]] = {
     # Tolerance value (rtol, atol)
-    # The current most relaxed values are for aten::matmul
+    # The current most relaxed values on f32 are for aten::matmul
```

Comment:
nit: fp32?

Reply:
I was using the rust notation (shorter). Changed.